packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -747,6 +748,18 @@ mixin WidgetInspectorService {
   /// The current [WidgetInspectorService].
   static WidgetInspectorService get instance => _instance;
   static WidgetInspectorService _instance = _WidgetInspectorService();
+
+  final ValueNotifier<bool> _isSelectMode = ValueNotifier<bool>(true);
+
+  /// Whether the inspector is in select mode.
+  ///
+  /// In select mode, pointer interactions trigger widget selection instead of
+  /// normal interactions. Otherwise the previously selected widget is
+  /// highlighted but the application can be interacted with normally.
+  ValueListenable<bool> get isSelectMode => _isSelectMode;
```

Comment:
Does this `ValueListenable` have to be public? Would it be enough to just have a public getter/setter for _isSelectMode.value?

Reply:
Actually, do we even need public setter/getters for this? Who would be calling this outside of this file? Looks like it is just a test? Should the setter even be marked as `@visibleForTesting`?